### ISSUE TITLE
Border is getting disappered for "Automatically check for updates" & "Update channel" dropdown upon getting the focus in High contrast black mode.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#348]
### Steps to Reproduce

Pre-Requisite: Change the system high contrast mode to black before launching the application.

1. Launch Xamarin Workbooks application.  Which invokes "Create New Workbook" window.
2. Navigate to Tools -> Preferences. Invoke preferences tab. It will open the general tab of preferences. Invoke the update tab item using mouse (As tab items are not accessible by keyboard)
3. Navigate to "Automatically check for updates" & "Update channel" dropdowns and observe the focus.

### Expected Behavior
Border should not disappear upon setting the focus.
### Actual Behavior
Border is getting disappered for "Automatically check for updates" & "Update channel" dropdown upon getting the focus.
### User Impact
User will be confused with loss of border
### Screenshots
![9 hc black](https://user-images.githubusercontent.com/22956130/40723335-598deab2-643c-11e8-839d-934a0b673d62.PNG)
![9 hc white](https://user-images.githubusercontent.com/22956130/40723336-59c45002-643c-11e8-8827-e54434686333.PNG)
![9 normal mode](https://user-images.githubusercontent.com/22956130/40723337-59f4f658-643c-11e8-98a6-c458c80f0dc0.PNG)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
